docs/examples/sock-shop.md

```diff
@@ -167,31 +200,22 @@ Examining this command, we see two new arguments:
 gitops add app \
 --url ssh://git@github.com/example/microservices-demo.git \
 --path ./deploy/kubernetes/helm-chart \
---app-config-url NONE \
+--app-config-url ssh://git@github.com/example/external.git
 --deployment-type helm \                   # (1)
 --helm-release-target-namespace sock-shop  # (2)
+--auto-merge
 ```
 
 1. Since we're pulling the chart from a git repository, we need to explicitly state that we're using a helm chart. If we were using a helm repository, we would use `--chart <chart name>` instead of `--path <path to application>` and the deployment type would be unambiguous
 2. The application will be deployed in the namespace specified by `--helm-release-target-namespace`
 
 
-## Deployment with Managed GitOps Resources
-As mentioned above, Weave GitOps allows you to also manage your GitOps resources via GitOps. This has several advantages:
-1. Increased control - you can alter parameters such as synchronization interval via updates to git
-2. Upgradability - when the Weave GitOps upgrades can be managed via git updates
-3. Reviewability and Auditability - Changes performed via git can be tracked; additionally, the default behavior of `gitops add app` is to create pull requests for upstream repositories which makes reviewing and auditing straightforward using the same tools used during development
-4. Recoverability - if the cluster is restored after failure, the management resources can be restored from git
+### Deployment with GitOps Resources in Application Repository
 
-To run with managed GitOps resources, we need to tell Weave GitOps where to put them. The default behavior of putting them in a private directory within the application repository works fine for repositories under our control (but doesn't work if we want to use either a helm repository or an open source git repository). To use the default, we simply leave off the `--app-config-url NONE` parameter (or, equivalently, use `--app-config-url ''`). This will store the manifests for our GitOps resources in the `.wego` directory within our application repository.
+As we mentioned above, it's possible to have a single repository perform hold both the application and the configuration. If you place the application manifests in the configuration repository passed to `gitops install`, you can leave off the separate `--app-config-url` parameter. In this case, we would either have had to pass the `microservices-demo` URL to `gitops install` or copy the application manifests into the `external` repository. Let's proceed as if we had initialized the cluster with: `gitops install --app-config-url ssh://git@github.com/example/microservices-demo.git`.
```

Comment:
I would break the install step out explicitly in a new line to make it more eye-grabbing.

So replacing:
> Let's proceed as if we had initialized the cluster with: `gitops install --app-config-url ssh://git@github.com/example/microservices-demo.git`

with:
> This section will show the steps following the initialization of a cluster with `gitops install --app-config-url ssh://git@github.com/example/microservices-demo.git`

Reply:
Looks like it has been updated. Resolving.